integration/client/import_test.go

```diff
@@ -84,7 +84,7 @@ func testExportImport(t *testing.T, imageName string) {
 		t.Fatal(err)
 	}
 
-	dstFile, err := os.CreateTemp("", "export-import-test")
+	dstFile, err := os.CreateTemp(t.TempDir(), "export-import-test")
```

Comment:
Please remove the `defer` lines 91-94

Reply:
I totally agree with removing the defer'ed `os.Remove*()` calls, but I feel like the `os.Close()` or `.Close()` deferred calls are still needed, otherwise the temporary file resource will be left open. Am I missing something here? I'm fairly novice at programming in golang compared to C/C++, python, etc, so I could be missing something key.

Reply:
☝️ assumes that the consumer of the file object doesn't handle closing the object, of course (the joy of borrowing/passing around resources).

Reply:
Keeping `.Close()` would be reasonable, though I'd recommend doing that in a `t.Cleanup` instead of a `defer`.

Reply:
Same comment about using `t.TempDir()` inside an `os.CreateTemp()`